src/log/log_slot.c

```diff
@@ -262,7 +262,7 @@ __wt_log_slot_new(WT_SESSION_IMPL *session)
 		/*
 		 * If we didn't find any free slots signal the worker thread.
 		 */
-		(void)__wt_cond_auto_signal(session, conn->log_wrlsn_cond);
+		WT_RET(__wt_cond_auto_signal(session, conn->log_wrlsn_cond));
```

Comment:
This looks dangerous to me. If {{__wt_log_slot_new}} returns an error {{__log_slot_switch_internal}} will return with {{WT_MYSLOT_CLOSE}} set which feels broken. @sueloverso do you think this is an OK change?